bundles/net.tourbook/src/net/tourbook/importdata/RawDataManager.java

```diff
@@ -1504,6 +1538,239 @@ public void clearInvalidFilesList() {
+
+            // loop: all selected tours in the viewer
+            for (final Long selectedTourId : selectedTourIds) {
+
+               monitor.worked(1);
+               monitor.subTask(NLS.bind(
+                     Messages.Import_Data_Dialog_Reimport_SubTask,
+                     new Object[] { ++imported, importSize }));
+
+               final TourData tourData = TourManager.getTour(selectedTourId);
+
+               if (tourData == null) {
+                  continue;
+               }
+
+               deleteTourValuesFromTour(tourValueTypes, tourData);
+
+               if (monitor.isCanceled()) {
+
+                  // user has canceled the deletion -> ask if the whole deletion should be canceled
+
+                  final boolean[] isCancelDeletion = { false };
+
+                  display.syncExec(() -> {
+
+                     if (MessageDialog.openQuestion(display.getActiveShell(),
+                           Messages.Import_Data_Dialog_IsCancelTourValuesDeletion_Title,
+                           Messages.Import_Data_Dialog_IsCancelTourValuesDeletion_Message)) {
+
+                        isCancelDeletion[0] = true;
+
+                     }
+                  });
+
+                  if (isCancelDeletion[0]) {
+                     break;
+                  }
+               }
+            }
+
+            updateTourData_InImportView_FromDb(monitor);
+
+            // reselect tours, run in UI thread
+            display.asyncExec(tourViewer::reloadViewer);
+         }
+      };
+
+      try {
+         new ProgressMonitorDialog(Display.getDefault().getActiveShell()).run(true, true, importRunnable);
+      } catch (final Exception e) {
+         TourLogManager.logEx(e);
+      } finally {
+
+         final double time = (System.currentTimeMillis() - start) / 1000.0;
+         TourLogManager.addLog(//
+               TourLogState.DEFAULT,
+               String.format(RawDataManager.LOG_DELETED_END, time));
+      }
+   }
+
+   public void deleteTourValuesFromTour(final List<TourValueType> tourValueTypes,
+                                        final TourData tourData) {
+
+      final Long tourId = tourData.getTourId();
+
+      /*
+       * tour must be removed otherwise it would be recognized as a duplicate and therefore not
+       * imported
+       */
+      final TourData oldTourDataInImportView = _toursInImportView.remove(tourId);
+
+      TourData clonedTourData = null;
+      try {
+
+         clonedTourData = (TourData) tourData.clone();
+
+         // loop: For each tour value type, we save the associated data for future display
+         //to compare with the new data
+         for (final TourValueType tourValueType : tourValueTypes) {
+
+            switch (tourValueType) {
+
+            case TIME_SLICES_TIME:
+               for (int index = 0; index < tourData.timeSerie.length; ++index) {
+                  tourData.timeSerie[index] = 0;
+               }
```

Comment:
It is useful for me. For old tours, my watch didn't have GPS but heart rate and time. I added in MT a GPX file that has time data but I want to remove the time data as otherwise, MT thinks I have cycled/ran 2h instead of 1h (1h for my watch file and 1h from the GPX)

Reply:
The title of the group is `Tour Values to Delete` which would be wrong for time data. I think it is better to create a new group with the title `Reset Tour Values` which contains time data

Reply:
Doesn't `Reset `mean the same thing than `Delete `? I am not sure a new category for one data makes sense

Reply:
`Delete` is for me, to remove it, e.g. for a data serie by setting it to `null`, which needs no space anymore, but you are not removing the data, you just set all values to 0

>I am not sure a new category for one data makes sense

For one data, it could be displayed without a group, but the group title tells the user the action, without a group title you have to write the action also into the checkbox text

Reply:
Fixed
![image](https://user-images.githubusercontent.com/25235665/115628508-45184000-a2f0-11eb-9503-c21c3834d17e.png)

Reply:
Perfect

Maybe a tooltip or info label should explain shortly what action is done in reset and delete. As you can see in some pref pages I prefer to use an info lable because a tooltip must first be hovered to see it.

When implementing #333 then "Only the selected tours" should be at the top of the radio button and always preselected because the other 2 options must be enabled first with an unlock button

Reply:
Fixed
![image](https://user-images.githubusercontent.com/25235665/115748212-7bec6580-a385-11eb-9b5d-c70be53f95e2.png)